script.js

```diff
@@ -1,66 +1,73 @@
+import Books from './books.js';
+
 const addTitle = document.getElementById('Title');
 const addAuthor = document.getElementById('Author');
 
-class Books {
-  constructor(title, author) {
-    this.title = title;
-    this.author = author;
-  }
+function getlocalStorage() {
+  return JSON.parse(localStorage.getItem('Book Details'));
 }
 
-const storedBooks = [
-  //   {
-  //     title: 'Title 1',
-  //     author: 'Author 1'
-  //   },
-  //   {
-  //     title: 'Title 2',
-  //     author: 'Author 2'
-  //   }
-];
+function setlocalStorage(store) {
+  localStorage.setItem('Book Details', JSON.stringify(store));
+  console.log(store);
+}
+
+class BookDetails {
+  constructor(book = []) {
+    this.storedBooks = book;
+  }
+
+  addBook = function (book) {
```

Comment:
- [ ] You can remove the function keyword because class methods don't need function keywords. It would be nice if you write it as `addBook(book) {` :blush:

Reply:
Thanks for the honest review, addBook(book) { is a very effective way writing method but it throws a linter  error ( Expected 'this' to be used by class generator method 'addBook()' ) . Therefore we use the function Keyword method. Again Thanks for the insights.

Reply:
This should work, maybe if you would need help reach out to me on slack by searching my name **Hammas Bin Saleem**